.changeset/afraid-beans-retire.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-theming": major
+---
+
+Make theming package public
```

Comment:
**suggestion:** This feels like a commit message. I feel for a "release note" it should be something more powerful like "First public release of theming package".